sdk/securityinsights/Azure.ResourceManager.SecurityInsights/src/autorest.md

```diff
@@ -34,6 +35,9 @@ rename-mapping:
   WatchlistItem.properties.updated: UpdatedOn
   AlertRule: SecurityInsightsAlertRule
   ThreatIntelligenceIndicatorModel: ThreatIntelligenceIndicator
+  EntityKindEnum: EntityKind
+  Entity: EntityData
+  Relation: IncidentRelation
```

Comment:
Well I believe this RP has not been polished right? But I still suggest we should prepend RP name on these becasue the new names here are still very generic

Reply:
fixed